### PR TITLE
Content changes

### PR DESCRIPTION
**Pull request by arh30**
Requested changes detailed on card

https://favro.com/organization/b03d6d6d9b11b61167ac4246/418c53a770389640d415a230?card=Dev-12686

### CODE REVIEW

File: htdocs/news.html [hunk 1]
```diff
@@ -44,17 +44,68 @@ <h1>News &amp; Materials</h1>
         <div class="container">
           <div class="news-post-holder">
 
+            <div class="news-post news-post-languages">
+              <div class="img-holder">
+                <div class="img-wrap">
+                  <img src="/static/images/openletter.jpg" alt="News" />
+                </div>
+                <div class="time-wrap"><a href="#">#News</a><time datetime="2019-04-23">9th July 2019</time></div>
```
Comment (rgoj) [inline htdocs/news.html]: value of `datetime` doesn't match the date

File: htdocs/news.html [hunk 2]
```diff
@@ -44,17 +44,68 @@ <h1>News &amp; Materials</h1>
...
+                <a href="/openletter.html">Read more</a></p>
+              </div>
+            </div>
+
+            <div class="news-post">
+              <div class="img-holder">
+                <div class="img-wrap">
+                  <img src="/static/images/news-17.jpg" alt="News" />
+                </div>
+                <div class="time-wrap"><a href="#">#News</a><time datetime="2019-04-23">30th June 2019</time></div>
```
Comment (rgoj) [inline htdocs/news.html]: value of `datetime` doesn't match the date

File: htdocs/news.html [hunk 3]
```diff
@@ -44,17 +44,68 @@ <h1>News &amp; Materials</h1>
         <div class="container">
           <div class="news-post-holder">
 
+            <div class="news-post news-post-languages">
+              <div class="img-holder">
+                <div class="img-wrap">
+                  <img src="/static/images/openletter.jpg" alt="News" />
```
Comment (rgoj) [inline htdocs/news.html]: alt="Large group of people posing for photo with colourful fight inequality banners" ?

File: htdocs/news.html [hunk 4]
```diff
@@ -44,17 +44,68 @@ <h1>News &amp; Materials</h1>
...
+                <strong class="title">OPEN LETTER: WORLD LEADERS MUST STOP FUELLING INEQUALITY</strong>
+                <p>This open letter is written by the growing global movement to fight inequality - to call out the failure of the current approach by governments and to set out a vision for radical change.
+                <a href="/openletter.html">Read more</a></p>
+              </div>
+            </div>
+
+            <div class="news-post">
+              <div class="img-holder">
+                <div class="img-wrap">
+                  <img src="/static/images/news-17.jpg" alt="News" />
```
Comment (rgoj) [inline htdocs/news.html]: alt text missing

File: htdocs/news.html [hunk 5]
```diff
@@ -44,17 +44,68 @@ <h1>News &amp; Materials</h1>
...
+                <strong class="title">Fight Inequality Alliance Global Gathering - June 2019 - Summary Report</strong>
+                <p>Fight Inequality Alliance held its fifth Global Gathering in Kitwe, Zambia at the Mindolo Ecumenical Foundation from 16-20 June 2019. This report summary will outline some of the critical outcomes of the meeting as we grow, mobilise and campaign to end inequality.
+                <a href="https://docs.google.com/document/d/1_wCnZyWC9WQ_ycbODKyPB1_u1n5ZiV-EdUU8OmbSi8Q/edit">Read more</a></p>
+              </div>
+            </div>
+
+            <div class="news-post">
+              <div class="img-holder">
+                <div class="img-wrap">
+                  <img src="/static/images/news-18.jpg" alt="News" />
```
Comment (rgoj) [inline htdocs/news.html]: alt text missing

File: htdocs/news.html [hunk 6]
```diff
@@ -44,17 +44,68 @@ <h1>News &amp; Materials</h1>
...
+                </div>
+                <div class="time-wrap"><a href="#">#News</a><time datetime="2019-04-23">24th June 2019</time></div>
```
Comment (rgoj) [inline htdocs/news.html]: value of datetime doesn't match the date

File: htdocs/news.html [hunk 7]
```diff
@@ -44,17 +44,68 @@ <h1>News &amp; Materials</h1>
...
                 </div>
-                <div class="time-wrap"><a href="/news/2019-04-10.html">#News</a><time datetime="2019-04-23">23 Apr 2019</time></div>
+                <div class="time-wrap"><a href="#">#News</a><time datetime="2019-04-23">23 Apr 2019</time></div>
                 <a href="/news/2019-04-23.html" class="btn">Read more</a>
               </div>
               <div class="text-holder">
                 <strong class="title"> Registrations open for 2019 Fight Inequality Alliance global gathering</strong>
-                <p>Allies will gather in Kitwe, Zambia, from 16-20 June for the annual alliance gathering of members and allies from around the world.  <a href="/news/2019-04-23.html">Read more</a></p>
+                <p>Allies will gather in Kitwe, Zambia, from 16-20 June for the annual alliance gathering of members and allies from around the world.
+                <a href="/news/2019-04-23.html">Read more</a></p>
```
Comment (rgoj) [inline htdocs/news.html]: 💯It's the little things like this link on a new line that make my heart sing in a PR review :)

File: htdocs/news.html [hunk 8]
```diff
@@ -44,17 +44,68 @@ <h1>News &amp; Materials</h1>
...
+                <strong class="title">The Pan-African Fight Inequality Alliance: Kitwe Declaration: building Pan-African solidarity</strong>
+                <p>In June 2019, social movements, labour movements, NGO representatives and activists from eleven African countries gathered in Zambia under the banner of the Pan-African Fight Inequality Alliance (FIA).
+                <a href="/news/2019-06-24.html">Read more</a></p>
+              </div>
+            </div>
+
             <div class="news-post">
               <div class="img-holder">
                 <div class="img-wrap">
                   <img src="/static/images/news.png" alt="News" />
```
Comment (rgoj) [inline htdocs/news.html]: Oh, I see, we never cared about alt text here :'(

File: htdocs/static/scss/layouts/_common.scss [hunk 9]
```diff
@@ -454,7 +465,7 @@ h2,
 
 .video-holder {
   position: relative;
-  
+
```
Comment (rgoj) [inline htdocs/static/scss/layouts/_common.scss]: Ah, satisfying :)

File: htdocs/news/2019-06-24.html [hunk 10]
```diff
@@ -0,0 +1,115 @@
...
+              <li>Calling on our governments to reject the historical debt of African nations to the international financial institutions, and recognise the social and environmental debts that those institutions and the governments that control them owe to us.</li>
+              <li>We will support mining-affected communities’ rights to a dignified life - free from exploitation, pollution, and exclusion.</li>
+              <li>Calling on our governments to put in place national budgets that prioritise the needs of women, persons living with disabilities, youths, indigenous groups and other marginalised groups.</li>
+            </ul>
+            <p>We pledge to work in solidarity with African people everywhere – for peace, justice, and dignity. Our struggles are connected and the future we want, the Africa we want, is ours to dream and make real. We are also part of a global alliance, recognising that our struggles will be compromised if it excludes struggles in other parts of the globe.</p>
+            <p>A Luta Continua!</p>
```
Comment (rgoj) [inline htdocs/news/2019-06-24.html]: From the text of the task, I understood that they wanted the image placed above this paragraph...?

Here's the text of the task:

> Post, under News & Materials, the FIA Global Gathering Summary Report (as a link to the google doc) and the Pan-Africa Declaration (as a page). For the Pan-Africa Declaration page, please use the photo, attached, placed before 'A Luta Continua' at the bottom.

or did I misunderstand something?

Comment (arh30) [reply htdocs/news/2019-06-24.html]: you're damn right, will get this added, thanks

File: htdocs/news.html (hunk 8 above)
Comment (arh30) [reply htdocs/news.html]: I've added alt texts to all the images on this page now, although i can't put the time into covering the whole site today :(

File: htdocs/news.html (hunk 6 above)
Comment (arh30) [reply htdocs/news.html]: good spot!

File: htdocs/news.html (hunk 8 above)
Comment (rgoj) [reply htdocs/news.html]: Small wins, awesome! :)

**Pull request merged**